src/NuGet.Core/NuGet.CommandLine.XPlat/Strings.resx

```diff
@@ -818,4 +818,32 @@ Non-HTTPS access will be removed in a future version. Consider migrating to 'HTT
   <data name="SourcesCommandValidProtocolVersion" xml:space="preserve">
     <value>The protocol version specified is invalid. Provide a valid protocol version.</value>
   </data>
+  <data name="pkgSearch_Description" xml:space="preserve">
+    <value>Searches a given source using the query string provided. If no sources are specified, all sources defined in the NuGet.Config are used.</value>
+  </data>
+  <data name="pkgSearch_ExactMatchDescription" xml:space="preserve">
+    <value>Return an exact match only as a search result.</value>
+  </data>
+  <data name="pkgSearch_InteractiveDescription" xml:space="preserve">
+    <value>Allows the command to stop and wait for user input or action (for example to complete authentication).</value>
+  </data>
+  <data name="pkgSearch_PrereleaseDescription" xml:space="preserve">
+    <value>Allow prerelease packages to be shown.</value>
+  </data>
+  <data name="pkgSearch_SkipDescription" xml:space="preserve">
+    <value>The number of results to skip, for pagination. The default is 0.</value>
+  </data>
+  <data name="pkgSearch_SourceDescription" xml:space="preserve">
+    <value>The package source to search. You can pass multiple `--source` options to search multiple package sources.</value>
```

Comment:
```suggestion
    <value>The package source to search. You can pass multiple `--source` options to search multiple package sources. Example: `--source https://api.nuget.org/v3/index.json`.</value>
```
Is this right, it's a URL? If so an example is good.

Note that `nuget list` allows semicolon separated sources, it looks like you're requiring repeated `--source` flags -- which is fine.

Reply:
I believe most of our `source` parameters refer to the source name, not the source URI. I recently thought whether the output should group by URI then show a comma-delimited list of source names that use that URI (note: this is kind of an odd scenario, to have the same source with different names): https://github.com/NuGet/Home/pull/12970#discussion_r1373759704

Regarding multiple sources, what do you think about enabling this option that allows an argument list to be  sent with a single option, rather than a semicolon separated approach?
` --opt 1 2 3`
https://learn.microsoft.com/en-us/dotnet/api/system.commandline.option.allowmultipleargumentspertoken#system-commandline-option-allowmultipleargumentspertoken

Reply:
That formatting of multiple arguments seems fine as long as it's in the command line help.

For names vs URI... I'm not sure I'm understanding how this works. URI to name mapping is in a nuget config, right? How do I choose a Nuget config? What if I don't have one, I just want to see where a package is? We do this all the time for tracking transport packages on internal feeds. What if I want to share this in a script, and your Nuget config has the exact functionality of mine but you chose different arbitrary feed names in the Nuget config?

Reply:
`--souce` can take both name and URI. if a name is specified, the name to URI mapping in the default config files for the working directory is used. If the mapping is not found, that is a "source not found" error. If a URI is specified, it will be used to as a source directly.

Reply:
Nevermind what I suggested about URI versus name. I didn't realize that config file wasn't an option here. Thanks for explaining, Nigusu!

> That formatting of multiple arguments seems fine as long as it's in the command line help.

Sounds good! Nigusu, similar to your idea in another thread, you could decide on enabling this in a separate PR.